src/constants/wallets.ts

```diff
@@ -265,6 +270,12 @@ export const wallets: Record<WalletType, WalletConfig> = {
     icon: GenericWalletIcon,
     connectionTypes: [WalletConnectionType.TestWallet],
   },
+  [WalletType.Privy]: {
+    type: WalletType.Privy,
+    stringKey: STRING_KEYS.KEPLR,
+    icon: GenericWalletIcon,
+    connectionTypes: [WalletConnectionType.Privy],
```

Comment:
This file defines all the different types of wallets and their connection types (e.g browser injection, or walletconnect etc). Here we are using a new connection type for Privy specifically. 
This connection type will be referenced later